src/sentry/integrations/slack/utils.py

```diff
@@ -246,8 +256,9 @@ def build_group_attachment(group, event=None, tags=None, identity=None, actions=
             footer += u" (+{} other)".format(len(rules) - 1)
 
     obj = event if event is not None else group
+    project = Project.objects.get_from_cache(id=obj.project_id)
```

Comment:
Here might be worth checking if the thing has been bound already, since this also might literally be referencing the same object that is above. And we're doing an unnecessary call if so.